packages/map-template/src/hooks/useMapBoundsDeterminer.js

```diff
@@ -92,7 +87,7 @@ const useMapBoundsDeterminer = () => {
 
                         getDesktopPaddingBottom().then(desktopPaddingBottom => {
                             setMapPositionKnown(kioskLocation.geometry);
-                            goToGeometry(mapType, kioskLocation.geometry, mapsIndoorsInstance, desktopPaddingBottom, 0, startZoomLevel, currentPitch, bearing);
+                            goto(kioskLocation.geometry, mapsIndoorsInstance, desktopPaddingBottom, 0, startZoomLevel, currentPitch, bearing);
```

Comment:
Should probably follow the casing to be `goTo` instead. 😄

Reply:
🫡